### PR TITLE
refactor: Replace now CLI with vercel CLI

### LINKED ISSUE #23: Now cli deprecated in favor of Vercel
Update docs and test new CLI for Vercel

### PR DESCRIPTION
**Pull request by adriansberg**
Closes #23

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,18 +16,18 @@ At the same time, an `.env`-file (ignored by git) will be created in the project
 
 ## Deployment
 
-This project is supports deployment to [Now](https://zeit.co/now). Add secrets from `.env`-file by using the `now`-CLI:
+This project is supports deployment to [Vercel](https://vercel.com/home). Add secrets from `.env`-file by using the `vercel`-CLI:
```
[the remark this answers is not in the document]
Comment (adriansberg) [reply README.md:19]: no u

File: package.json [hunk 2]
```diff
@@ -33,7 +33,6 @@
     "eslint-plugin-typescript": "^0.14.0",
     "jest": "^24.8.0",
     "nodemon": "^1.17.2",
-    "now": "^15.8.7",
```
Comment (cbrevik) [inline package.json:36]: Should `vercel` be added as a dev dep?

Comment (adriansberg) [reply package.json:36]: Definitely. Added now.

**Review by cbrevik — APPROVED** [review]
Woop woop 🦞

**Pull request merged**